src/UXClient/Models/Strings.ts

```diff
@@ -51,8 +51,8 @@ class Strings {
         "Next Month": "Next Month",
         "Unassigned Time Series Instances": "Unassigned Time Series Instances",
         "Search Globally": "Search Globally",
-        "Show More Instances": "Show More Instances",
-        "Show More Hierarchies": "Show More Hierarchies",
+        "Show More Instances": "Show more instances",
+        "Show More Hierarchies": "Show more hierarchies",
```

Comment:
i think "Show more hierarchies" should just be "Show more" since they are sort of intermediary nodes, not specifically hierarchies